src/alexandria3k/data_sources/uspto.py

```diff
@@ -637,6 +999,718 @@ def Column(self, col):
+                "type_of_correction",
+            ),
+            ColumnMeta(
+                "gazette_number",
+            ),
+            ColumnMeta(
+                "gazette_date",
+            ),
+            ColumnMeta(
+                "correction_text",
+            ),
+        ],
+    ),
+    TableMeta(
+        "usp_field_of_classification",
+        foreign_key="patent_id",
+        parent_name="us_patents",
+        primary_key="id",
+        cursor_class=PatentsDetailsCursor,
+        extract_multiple=all_getter(
+            "us-bibliographic-data-grant/us-field-of-classification-search"
+        ),
+        columns=[
+            ColumnMeta("id"),
+            ColumnMeta("container_id"),
+            ColumnMeta("patent_id"),
+            ColumnMeta(
+                "ipcr_classification",
+                getter("us-classifications-ipcr"),
+            ),
+            ColumnMeta(
+                "cpc_classification_text",
+                getter("classification-cpc-text"),
+            ),
+            ColumnMeta(
+                "cpc_classification_combination_text",
+                getter("classification-cpc-combination-text"),
+            ),
+            ColumnMeta(
+                "national_edition",
+                getter("classification-national/edition"),
+            ),
+            ColumnMeta(
+                "national_main",
+                getter("classification-national/main-classification"),
+            ),
+            ColumnMeta(
+                "national_further",
+                getter("classification-national/further-classification"),
+            ),
+            ColumnMeta(
+                "national_additional_info",
+                getter("classification-national/additional-info"),
+            ),
+            ColumnMeta(
+                "national_linked_code_group",
+                getter("classification-national/linked-indexing-code-group"),
+            ),
+            ColumnMeta(
+                "national_unlinked_code",
+                getter("classification-national/unlinked-indexing-code"),
+            ),
+            ColumnMeta(
+                "national_text",
+                getter("classification-national/text"),
+            ),
+        ],
+    ),
+    TableMeta(
+        "usp_inventors",
+        foreign_key="patent_id",
+        parent_name="us_patents",
+        primary_key="id",
+        cursor_class=PatentsDetailsCursor,
+        extract_multiple=alternative_path_getter(
+            "us-bibliographic-data-grant/us-parties/inventors/inventor",
+            "us-bibliographic-data-grant/parties/inventors",
+        ),
+        columns=[
+            ColumnMeta("id"),
+            ColumnMeta("container_id"),
+            ColumnMeta("patent_id"),
+            ColumnMeta(
+                "sequence",
+                agetter("sequence"),
+            ),
+            ColumnMeta(
+                "designation",
+                agetter("designation"),
+            ),
```

Comment:
Find a way to remove duplication between the schema definition of agents, applicants, assignees.  See how it's done in ORCID.